docs/source/reference/kubernetes/kubernetes-deployment.rst

```diff
@@ -231,7 +232,7 @@ Notes for specific Kubernetes distributions
 
 Some Kubernetes distributions require additional steps to set up GPU support.
 
-Rancher Kubernetes Engine 2 (RKE2)
+Rancher Kubernetes engine 2 (RKE2)
```

Comment:
`Rancher Kubernetes Engine` is the name of a specific thing, so we should keep it capitalized.